src/Parser/ComputeMenuCommandParser.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE InstanceSigs #-}
+module Parser.ComputeMenuCommandParser(parse, ParseError(..)) where
+import Data.ComputeMenuCommand (Command (..))
+import qualified Data.WordError as WordError(ParseError(..))
+import Util.ParseUtil (isCorrectWord)
+
+data ParseError = UnknownCommand String | WordParseError WordError.ParseError String deriving Eq
+
+instance Show ParseError where
+    show :: ParseError -> String
+    show (WordParseError err word) =  "Word '" <> word <> "'. " <> show err
+    show (UnknownCommand cmd) =  "Unsupported command: '" <> cmd <> "'."
```

Comment:
Here and everywhere else you want to concatenate strings, it's better to use `prinf` from `Text.Printf`. And in general, it's better to use `Text`